meson.build

```diff
@@ -399,30 +399,28 @@ if host_machine.system() == 'freebsd'
   endif
 endif
 
-efiboot = dependency('efiboot', required: get_option('plugin_uefi_capsule'))
-efivar = dependency('efivar', required: get_option('plugin_uefi_capsule'))
-if build_standalone and efiboot.found() and efivar.found()
-
+# EFI
+if build_standalone
   efi_app_location = join_paths(libexecdir, 'fwupd', 'efi')
   conf.set_quoted('EFI_APP_LOCATION', efi_app_location)
-
   if host_cpu == 'x86'
-          EFI_MACHINE_TYPE_NAME = 'ia32'
+    EFI_MACHINE_TYPE_NAME = 'ia32'
   elif host_cpu == 'x86_64'
-          EFI_MACHINE_TYPE_NAME = 'x64'
+    EFI_MACHINE_TYPE_NAME = 'x64'
   elif host_cpu == 'arm'
-          EFI_MACHINE_TYPE_NAME = 'arm'
+    EFI_MACHINE_TYPE_NAME = 'arm'
   elif host_cpu == 'aarch64'
-          EFI_MACHINE_TYPE_NAME = 'aa64'
+    EFI_MACHINE_TYPE_NAME = 'aa64'
   else
-          EFI_MACHINE_TYPE_NAME = ''
+    EFI_MACHINE_TYPE_NAME = ''
   endif
   conf.set_quoted('EFI_MACHINE_TYPE_NAME', EFI_MACHINE_TYPE_NAME)
+endif
 
+if build_standalone and get_option('plugin_uefi_capsule').enabled()
   if get_option('efi_binary')
     efi_binary = dependency ('fwupd-efi', fallback: ['fwupd-efi', 'fwupd_efi_dep'])
   endif
```

Comment:
I think this block is missing the disable auto if stuff. how about doing this instead:
```
uefi_capsule = get_option('plugin_uefi_capsule').disable_auto_if(host_machine.system() not in ['linux', 'freebsd']).allowed()
if build_standalone and uefi_capsule
```

Then use `if uefi_capsule` in the plugins folder

Reply:
Or move this whole block into the plugins folder

Reply:
> Or move this whole block into the plugins folder

That's probably the best thing to do; done -- lets see what CI thinks.